sonar-project.properties

```diff
@@ -6,4 +6,4 @@ sonar.host.url=https://sonarcloud.io
 sonar.organization=project-sunbird
 sonar.projectVersion=1.0
 sonar.exclusions=**/*.spec.js,**/libs/*.js,**/deploy/**,**/server/**,**/migration/*task.js,**/backward-compatibility.js,**/base-plugin.js,**/bootstrap-editor.js,**/ce-config.js,**/content-editor-api.js,**/content-editor.js,**/webpack.plugin.config.js,**/webpack.config.js,**/karmaconf.pluginframework.js,**/karmaconf.js,**/handle-empty-css.css,**/gulpfile.js,**/Gruntfile.js,**/app.js,**/test_framework/**,**/test/**
-sonar.javascript.lcov.reportPaths=/root/content-editor/coverage_framework/lcov/HeadlessChrome 78.0.3904 (Mac OS X 10.12.6)/lcov.info,/root/content-editor/coverage/lcov/HeadlessChrome 78.0.3904 (Mac OS X 10.12.6)/lcov.info
\ No newline at end of file
+sonar.javascript.lcov.reportPaths=coverage/lcov.info,tmp/coverage/lcov.info,/home/circleci/content-editor/coverage/lcov/HeadlessChrome\ 59.0.3071\ \(Linux\ 0.0.0\)/lcov.info
```

Comment:
change the path HeadlessChrome\ 59.0.3071\ \(Linux\ 0.0.0\.  this is not generic . it may change in future based on chrome headless version

Reply:
it just a test i was doing, actually its "coverage/lcov.info", doing the change